monitoring/prometheus/alerts/ceph_default_alerts.yml

```diff
@@ -214,6 +214,17 @@ groups:
             will be full in less than 5 days assuming the average fill-up
             rate of the past 48 hours.
 
+      - alert: Different MTU Size
```

Comment:
I'd rather use the term 'MTU Mismatch' (it's [well established](https://www.google.com/search?q=mtu+mismatch)) or 'MTU Size Mismath' to put emphasis on the size issue.